dev/com.ibm.ws.jdbc_fat_db2/fat/src/com/ibm/ws/jdbc/fat/db2/SQLJTest.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
```

Comment:
Need to update copyright year (you can add this as a new commit with just the text change without having to do another personal build)

Reply:
Done, thanks for catching